.gitignore

```diff
@@ -1,6 +1,7 @@
 database.py
 config.txt
 settings.py
+.vscode/launch.json
```

Comment:
I can see that .vscode/launch.json is still there🐱. You can stop tracking it with "git rm <file_path> --cached" add, commit, push 🚀

Reply:
thank you! <3